### PR TITLE
Ta 491 animation check improvement

### PR DESCRIPTION
**Pull request by Saptortacus**

### CODE REVIEW

File: module_constants.py [hunk 1]
```diff
@@ -243,6 +243,8 @@
 #Slot to keep agent_id of the killer
 slot_scene_prop_corpse_owner          = 60
 
+slot_scene_prop_sitting_agent         = 55
+
```
Comment (Thomas-Smyth) [inline module_constants.py]: Duplicated slot IDs

File: module_scripts.py [hunk 2]
```diff
@@ -14619,6 +14615,46 @@ def animation_menu_entry(string_id, **kwargs):
...
+          (agent_get_slot, ":x", ":agent_id", slot_agent_position_animation_x),
+          (agent_get_slot, ":y", ":agent_id", slot_agent_position_animation_y),
+          (position_set_x, pos11, ":x"),
+          (position_set_y, pos11, ":y"),
+            
+          (position_set_z_to_ground_level, pos10),
+          (position_set_z_to_ground_level, pos11),
+            
+          (get_distance_between_positions, ":distance", pos10, pos11),
+          (le, ":distance", 15),
```
Comment (Thomas-Smyth) [inline module_scripts.py]: Are the agent position slots even needed anymore? If the animation cancels when you move can you not simply find what animation the player is doing and then just assume that these checks need to be run?

**Comment by Saptortacus** [comment]
I applied the suggestion you made.
I can't see which slots have the same values though. But I removed those slots anyway as they no longer will be used.

**Comment by Thomas-Smyth** [comment]
Yeah, that looks better! Seems much neater, though give it a good test. I've experienced in the past that occasionally the returned animation isn't always the correct animation.

As with the duplicated slot IDs:
```
slot_scene_prop_sitting_agent         = 55
slot_scene_prop_store_ammo          = 55
```

**Comment by Thomas-Smyth** [comment]
The changes I've seen are good. If it passes testing then I'm happy for this to be merged.

**Pull request merged**